spec/controllers/admin/posts_controller_spec.rb

```diff
@@ -44,10 +44,20 @@
 
   describe 'POST create' do
     describe 'with valid params' do
-      it 'creates a new Admin::Post' do
-        expect {
-          post :create, post: valid_attributes
-        }.to change(Post, :count).by(1)
+      let(:file1) do
+        Rack::Test::UploadedFile.new('spec/support/1.jpg', 'image/jpeg')
+      end
+
+      it 'creates a new Admin::Post with photo' do
+        params = {
+          post: {
+            title: 'NewPostTitle',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.